src/main/java/competition/subsystems/shooterwheel/ShooterWheelSubsystem.java

```diff
@@ -2,44 +2,72 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import com.revrobotics.ControlType;
 
 import competition.IdealElectricalContract;
 import xbot.common.command.BaseSubsystem;
-import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.controls.actuators.XCANSparkMax;
+
 import xbot.common.injection.wpi_factories.CommonLibFactory;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.PropertyFactory;
 
 @Singleton
 public class ShooterWheelSubsystem extends BaseSubsystem {
     
-    final DoubleProperty spinWheelPowerProp;
-    private IdealElectricalContract contract;
-    public XCANTalon shooterWheelMaster;
+    final DoubleProperty targetRpmProp;
+    final DoubleProperty currentRpmProp;
+    public XCANSparkMax neoMasterMotor;
```

Comment:
the new politically correct version of this is leader/follower instead of master/slave